sources/US/CO/bighornsheep.json

```diff
@@ -1,16 +1,18 @@
 {
-	"name" : "CO Bighorn Sheep Hunt Units",
-        "country": "US",
-        "state": "CO",
-        "attribution": "Colorado Parks and Wildlife",
-    	"properties": {
-        	"id": "BSGMU",
-        	"name": "GMU_NAME"
-    },
-        "species": [
-		"bighorn sheep"
-	],
-        "filetype": "shp",
-        "url": "https://data.openbounds.org.s3.amazonaws.com/USAHunting/manual-downloads/CPW_BighornUnitBoundaries.ZIP"
+   "name" : "CO Game Management Units - Bighorn Sheep",
+   "url": "https://data.openbounds.org.s3.amazonaws.com/USAHunting/manual-downloads/co-BighornGMUBoundaries08152017.zip",
+   "filenameInZip": "BighornGMUBoundaries08152017.shp",
+   "species": [
+      "bighorn sheep"
+   ],
+   "attribution": "Colorado Parks and Wildlife",
+   "properties": {
+      "id": "BSGMU",
```

Comment:
property BSHGMU not found in source feature error processing feature.
properties: OrderedDict([(u'BSGMU', u'S44'), (u'GMU_NAME', u'BASALT'), (u'DAU_NAME', u'BASALT'), (u'HUNTING', u'YES'), (u'BSDAU', u'RBS-35'), (u'ACRES', 100062.30644), (u'INPUT_ID', u''), (u'INPUT_DATE', None), (u'EDITOR_NAM', u''), (u'EDIT_DATE', None), (u'Shape_STAr', 404939407.213), (u'Shape_STLe', 134467.915161)]